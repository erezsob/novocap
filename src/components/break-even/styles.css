.container {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    align-items: center;
    height: 20px;
    width: 90%;
    padding-top: 20px;
    color: white;
    font-weight: bold;
    font-size: 20px;
}

.barContainer {
    margin: 20px;
    width: 150px;
    height: 150px;
    position: relative;
    display: flex;
    flex-direction: column;
    justify-content: center;
}

.title {
    display: flex;
    flex-direction: column;
    padding-top: 13px;
}

.date {
    font-size: 11px;
    font-weight: 300;
}

.addCosts {
    color: #fff;
    font-size: 19px;
}