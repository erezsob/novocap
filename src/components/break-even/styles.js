import { blueMain } from '../../common/variables.css';

export default {
  card: {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    boxShadow: 'none',
    height: 208,
    backgroundColor: blueMain,
    cursor: 'pointer'
  },
  iconButton: {
    color: 'white',
    width: 24
  }
};
