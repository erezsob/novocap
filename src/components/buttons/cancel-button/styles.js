import { pinkyRed } from '../../../common/variables.css';

export default {
  cancel: {
    color: pinkyRed
  }
};
