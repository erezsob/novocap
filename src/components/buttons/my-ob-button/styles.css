@value blueLight, bpXSmall from '../../../common/variables.css';

.button {
    border-radius: 4px;
    background-color: blueLight;
    color: white;
    padding: 8px 0;
    font-size: 12px;
    display: table;
    border: 0;
    outline: 0;
    cursor: pointer;
}

.bigger {
    padding: 10px 0;
    font-size: 16px;
}

.icon,
.text {
    vertical-align: middle;
    text-align: center;
}

.text {
    padding: 0 20px;
    font-weight: lighter;
}

.icon ~ .text {
    display: none;
}

.icon {
    display: table-cell;
    padding: 0 10px;
    position: relative;
    top: 2px;
}

@media (min-width: bpXSmall) and (min-device-width: bpXSmall) {
    .icon ~ .text {
        padding-left: 0;
        display: table-cell;
    }
}
