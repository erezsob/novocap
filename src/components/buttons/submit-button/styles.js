import { greenGrass } from '../../../common/variables.css';

export default {
  submit: {
    color: greenGrass
  }
};
