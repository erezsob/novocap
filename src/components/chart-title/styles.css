.label {
    letter-spacing: 0.05em;
    font-size: 20px;
    border-bottom: 1px solid #efefef;
    padding: 20px;
    margin-bottom: 20px;
    font-weight: bold;
    color: #323435;
}