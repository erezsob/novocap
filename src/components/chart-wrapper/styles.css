.container {
    padding: 0 20px 30px;
    position: relative;
    font-size: 12px;
}