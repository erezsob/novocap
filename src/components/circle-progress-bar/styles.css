@value grayStatic, greenAtlantis, blueDark from '../../common/variables.css';

.percent {
    position: absolute;
    font-size: 30px;
    color: white;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
}

.chart {
    position: relative;
}

.successBar {
    fill: greenAtlantis
}

.progressBar {
    fill: blueDark
}

.staticBar {
    fill: grayStatic
}