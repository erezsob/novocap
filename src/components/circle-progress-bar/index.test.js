import React from 'react';
import { shallow, mount } from 'enzyme';
import renderer from 'react-test-renderer';
import { RadialChart } from 'react-vis';
import CircleProgressBar from './index';
import styles from './styles.css';

describe('<CircleProgressBar />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<CircleProgressBar percent={15} />);
  });

  it('renders', () => {
    expect(wrapper).toHaveLength(1);
  });

  it('renders properly', () => {
    const tree = renderer.create(<CircleProgressBar percent={15} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders a <RadialChart /> with the right properties', () => {
    expect(wrapper.find(RadialChart)).toHaveLength(1);
    expect(wrapper.find(RadialChart).props().animation).toBeTruthy();
  });

  it('renders with the right css classes', () => {
    wrapper = mount(<CircleProgressBar percent={15} />);
    expect(wrapper.find(`.${styles.percent}`)).toHaveLength(1);
    expect(wrapper.find(`.${styles.successBar}`)).toHaveLength(2);
    expect(wrapper.find(`.${styles.progressBar}`)).toHaveLength(2);
    expect(wrapper.find(`.${styles.staticBar}`)).toHaveLength(2);
  });
});
