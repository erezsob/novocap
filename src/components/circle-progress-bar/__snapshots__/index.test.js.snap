// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CircleProgressBar /> renders properly 1`] = `
<div>
  <div
    className="rv-xy-plot chart rv-radial-chart"
    style={
      Object {
        "height": "150px",
        "width": "150px",
      }
    }
  >
    <svg
      className="rv-xy-plot__inner"
      height={150}
      onClick={[Function]}
      onDoubleClick={[Function]}
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseMove={[Function]}
      onWheel={[Function]}
      style={undefined}
      width={150}
    >
      <g
        className="rv-xy-plot__series--arc__animation-wrapper"
      >
        <g
          className="rv-xy-plot__series rv-xy-plot__series--arc chart"
          onClick={[Function]}
          onContextMenu={[Function]}
          onMouseOut={[Function]}
          onMouseOver={[Function]}
          opacity={1}
          pointerEvents="none"
          transform="translate(75,75)"
        >
          <path
            className="rv-xy-plot__series rv-xy-plot__series--arc-path rv-radial-chart__series--pie__slice staticBar"
            d="M56.63118960624632,-41.144967660473114A70,70,0,1,1,4.9313697987961557e-14,-70L4.226888398968133e-14,-60A60,60,0,1,0,48.54101966249685,-35.26711513754838Z"
            onClick={[Function]}
            onContextMenu={[Function]}
            onMouseOut={[Function]}
            onMouseOver={[Function]}
            style={
              Object {
                "fill": 0,
                "opacity": 1,
                "stroke": 0,
              }
            }
          />
          <path
            className="rv-xy-plot__series rv-xy-plot__series--arc-path rv-radial-chart__series--pie__slice progressBar"
            d="M4.2862637970157365e-15,-70A70,70,0,0,1,56.63118960624632,-41.144967660473114L48.54101966249685,-35.26711513754838A60,60,0,0,0,3.67394039744206e-15,-60Z"
            onClick={[Function]}
            onContextMenu={[Function]}
            onMouseOut={[Function]}
            onMouseOver={[Function]}
            style={
              Object {
                "fill": 1,
                "opacity": 1,
                "stroke": 1,
              }
            }
          />
          <path
            className="rv-xy-plot__series rv-xy-plot__series--arc-path rv-radial-chart__series--pie__slice successBar"
            d="M4.2862637970157365e-15,-70L3.67394039744206e-15,-60Z"
            onClick={[Function]}
            onContextMenu={[Function]}
            onMouseOut={[Function]}
            onMouseOver={[Function]}
            style={
              Object {
                "fill": 2,
                "opacity": 1,
                "stroke": 2,
              }
            }
          />
        </g>
        <g
          className="rv-xy-plot__series rv-xy-plot__series--arc chart"
          onClick={[Function]}
          onContextMenu={[Function]}
          onMouseOut={[Function]}
          onMouseOver={[Function]}
          opacity={0}
          pointerEvents="all"
          transform="translate(75,75)"
        >
          <path
            className="rv-xy-plot__series rv-xy-plot__series--arc-path rv-radial-chart__series--pie__slice staticBar"
            d="M56.63118960624632,-41.144967660473114A70,70,0,1,1,4.9313697987961557e-14,-70L4.226888398968133e-14,-60A60,60,0,1,0,48.54101966249685,-35.26711513754838Z"
            onClick={[Function]}
            onContextMenu={[Function]}
            onMouseOut={[Function]}
            onMouseOver={[Function]}
            style={
              Object {
                "fill": 0,
                "opacity": 1,
                "stroke": "red",
              }
            }
          />
          <path
            className="rv-xy-plot__series rv-xy-plot__series--arc-path rv-radial-chart__series--pie__slice progressBar"
            d="M4.2862637970157365e-15,-70A70,70,0,0,1,56.63118960624632,-41.144967660473114L48.54101966249685,-35.26711513754838A60,60,0,0,0,3.67394039744206e-15,-60Z"
            onClick={[Function]}
            onContextMenu={[Function]}
            onMouseOut={[Function]}
            onMouseOver={[Function]}
            style={
              Object {
                "fill": 1,
                "opacity": 1,
                "stroke": "red",
              }
            }
          />
          <path
            className="rv-xy-plot__series rv-xy-plot__series--arc-path rv-radial-chart__series--pie__slice successBar"
            d="M4.2862637970157365e-15,-70L3.67394039744206e-15,-60Z"
            onClick={[Function]}
            onContextMenu={[Function]}
            onMouseOut={[Function]}
            onMouseOver={[Function]}
            style={
              Object {
                "fill": 2,
                "opacity": 1,
                "stroke": "red",
              }
            }
          />
        </g>
      </g>
    </svg>
  </div>
  <b
    className="percent"
  >
    15%
  </b>
</div>
`;
